azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/implementation/ApplicationGatewaysImpl.java

```diff
@@ -109,7 +109,7 @@ public void stop(Collection<String> applicationGatewayResourceIds) {
     @Override
     public Observable<String> stopAsync(Collection<String> applicationGatewayResourceIds) {
         if (applicationGatewayResourceIds == null) {
-            Observable.empty();
```

Comment:
added missing return type

Reply:
[P0]: this is important and must be fixed.